src/components/VmDetail/index.js

```diff
@@ -102,10 +98,12 @@ class VmDetail extends Component {
           <dd>{vm.get('fqdn')}</dd>
           <dt><span className='pficon pficon-screen' /> Console</dt>
           <VmConsoles vm={vm} onConsole={onConsole} />
-          <dt><span className='fa fa-database' /> Disks
-            &nbsp;
+          <dt><span className='fa fa-database' /> Disks&nbsp;
             <small>
-              (<a href='#' onClick={onToggleRenderDisks}>{this.state.renderDisks ? 'hide' : 'show'}</a>)
+              ({hasDisks
+              ? (<a href='#' onClick={onToggleRenderDisks}>{this.state.renderDisks ? 'hide' : 'show'}</a>)
```

Comment:
The surrounding `(` `)` are intended for rendering, so I'll keep them.
Double-space added